Package.swift

```diff
@@ -63,64 +71,100 @@ let interfaceBuildSettings: [CSetting] = [
     .unsafeFlags(["-I/usr/lib/swift"], .when(platforms: [.linux, .android])) // dispatch
 ]
 
+let swiftBuildSettings: [SwiftSetting] = [
+    .define("DEPLOYMENT_RUNTIME_SWIFT"),
+    .define("SWIFT_CORELIBS_FOUNDATION_HAS_THREADS"),
+]
+
+#if os(macOS)
+let products : [PackageDescription.Product] = [
+    .library(name: "SwiftFoundation", targets: ["Foundation"]),
+    .library(name: "SwiftFoundationXML", targets: ["FoundationXML"]),
+    .library(name: "SwiftFoundationNetworking", targets: ["FoundationNetworking"]),
+    .executable(name: "plutil", targets: ["plutil"]),
+]
+
+let foundationName = "SwiftFoundation"
+let foundationXMLName = "SwiftFoundationXML"
+let foundationNetworkingName = "SwiftFoundationNetworking"
```

Comment:
Do we still need to do this if we are no longer build on macOS? Mainly if we decide to use these names Cmake needs to match as well.

Reply:
No, I think I will revert this part.